python/Makefile

```diff
@@ -19,6 +19,9 @@ install:
 	pip install poetry
 	poetry install -E pyarrow -E hive -E s3fs
 
+check-license:
+	./dev/check-license
```

Comment:
is there a way to re-use the `check-license` file from `dev` in the root folder of the project?

Reply:
I prefer to keep it separate if we want to move pyiceberg to a different repo. It is a slimmed-down version of the original file that doesn't write the result into an output file, and doesn't accept an exclusion file (not needed :)